client/db.go

```diff
@@ -266,6 +266,27 @@ func (db *DB) DelRange(begin, end interface{}) Result {
 	return runOne(db, db.B.DelRange(begin, end))
 }
 
+// AdminMerge merges the range containing key and the subsequent
+// range. After the merge operation is complete, the range containing
+// key will contain all of the key/value pairs of the subsequent range
```

Comment:
Maybe. We don't seem to follow any convention here. I'm going to leave as-is for now.